webpack.config.js

```diff
@@ -0,0 +1,66 @@
+const path = require("path");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const { VueLoaderPlugin } = require("vue-loader");
+
+const isProduction = process.env.NODE_ENV == "production";
+
+const stylesHandler = MiniCssExtractPlugin.loader;
+
+function resolve (dir) {
+  return path.join(__dirname, dir)
+}
+
+const config = {
+  entry: [
+    resolve('./components/index.js')
+  ],
+  output: {
+    path: path.resolve(__dirname, "dist"),
+    filename: 'demosplan-ui.umd.js',
+    library: {
+      type: 'umd'
+    },
+  },
+  resolve: {
+    extensions: ['.js', '.vue'],
+    alias: { '@DpJs': path.resolve(__dirname, "../client/js") }
```

Comment:
the alias can be removed as i assume it does not work, and is gonna be obsolete when we initialize the boilerplate store within demosplan-core.

Reply:
thanks for the spot. this has been removed here [e73c1b7](https://github.com/demos-europe/demosplan-ui/pull/12/commits/e73c1b70fa6b9e1a568353381a7ada44f65dd62e)